checker/tests/nullness/UtilArrays.java

```diff
@@ -0,0 +1,41 @@
+import org.checkerframework.checker.nullness.qual.Nullable;
```

Comment:
This is an expanded version of https://github.com/eisop/jdk/pull/82#issuecomment-1949039533